### PR TITLE
Validate plugin names passed by user

### PR DESCRIPTION
**Pull request by chivay**
Fail when user tries to enable or disable plugin that doesn't exist.

### CODE REVIEW

**Comment by icedevml** [comment]
@drakvuf-jenkins This is OK to test.

**Comment by icedevml** [comment]
@drakvuf-jenkins Retest this please.

File: src/main.cpp [hunk 1]
```diff
@@ -137,16 +144,22 @@ static inline void disable_all_plugins(bool* plugin_list)
         plugin_list[i] = false;
 }
 
-static inline void enable_plugin(char* optarg, bool* plugin_list, bool* disabled_all)
+static inline int enable_plugin(char* optarg, bool* plugin_list, bool* disabled_all)
```
Comment (tklengyel) [inline src/main.cpp:147]: I prefer `bool` for such uses

Comment (chivay) [reply src/main.cpp]: Done

**Pull request merged**